app/services/summary_service.rb

```diff
@@ -75,10 +75,26 @@ def self.summary(summary_type, prison, page, params)
       summary.page_count = page_count
     }
   end
-# rubocop:enable Metrics/CyclomaticComplexity
-# rubocop:enable Metrics/LineLength
-# rubocop:enable Metrics/MethodLength
-# rubocop:enable Metrics/PerceivedComplexity
+  # rubocop:enable Metrics/CyclomaticComplexity
+  # rubocop:enable Metrics/LineLength
+  # rubocop:enable Metrics/MethodLength
+  # rubocop:enable Metrics/PerceivedComplexity
+
+  def self.entered_prison_dates(offender_list)
+    return if offender_list.empty?
+
+    offenders = offender_list.map(&:offender_no)
+
+    return if offenders.nil?
+
+    omic_start_date = Date.new(2019, 2, 4)
```

Comment:
I think we have a constant for this date somewhere complete with a comment?

Reply:
That would certainly help with POM-290. The start dates look like they may end up being different for England and Wales, so it will end up being prison-dependent - so might want to put the constant somewhere other than summary or handoverdate services.

Reply:
Happy to do this in a separate PR